packages/backend-defaults/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@backstage/backend-defaults",
+  "description": "Backend defaults used by Backstage backend apps",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts",
+    "alphaTypes": "dist/index.alpha.d.ts"
```

Comment:
Will we be using this from the start?

Reply:
I don't think we will, I figured it wouldn't hurt to have this here right?